src/main/java/com/telefonica/iot/cygnus/backends/ckan/CKANBackendImpl.java

```diff
@@ -259,7 +262,9 @@ private void insert(String recvTime, String resourceId, Map<String, String> attr
             urlPath = "/api/3/action/datastore_upsert";
         
             // do the CKAN request
-            CKANResponse res = requester.doCKANRequest("POST", urlPath, jsonString);
+            ArrayList<Header> headers = new ArrayList<Header>();
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/5f3e90c696b5b18965d514839a077d242388dd52